tests/functional/adapter/test_backup_table.py

```diff
@@ -0,0 +1,78 @@
+import pytest
+
+from dbt.tests.util import run_dbt
+
+
+_MODELS = {
+    "backup_is_false.sql": "{{ config(materialized='table', backup=False) }} select 1 as my_col",
+    "backup_is_true.sql": "{{ config(materialized='table', backup=True) }} select 1 as my_col",
+    "backup_is_undefined.sql": "{{ config(materialized='table') }} select 1 as my_col",
+    "backup_is_true_view.sql": "{{ config(materialized='view', backup=True) }} select 1 as my_col",
+    "syntax_with_distkey.sql": "{{ config(materialized='table', backup=False, dist='my_col') }} select 1 as my_col",
+    "syntax_with_sortkey.sql": "{{ config(materialized='table', backup=False, sort='my_col') }} select 1 as my_col",
+}
+
+
+class BackupTableBase:
+
+    @pytest.fixture(scope="class", autouse=True)
+    def run_dbt_results(self, project):
+        yield run_dbt(["run"])
```

Comment:
1. `["run"]` isn't needed
2. `yield` is a super overloaded term in pytest so it'd be rad to have a quick comment explaining what you're doing with this function

Reply:
1. I like being explicit, especially in tests
2. I actually don't think there's a difference between `yield` and `return` here since there's no teardown. I'm used to just using `yield` in this situation because of pytest. The only reason I need the results of `run_dbt()` is for the test below that we seemed to agree should be removed. So I'll remove that test and not return anything from the fixture.